### PR TITLE
New Gitkraken Cheat Sheet

### PR DESCRIPTION
**Pull request by mustang25**
## Description of new Instant Answer, or changes
This instant answer provides a cheat sheet for Gitkraken. 

## Testing & Review
To be completed by Community Leader (or DDG Staff) when reviewing Pull Request

**Pull Request**
- [ ] Title follows correct format (Specifies Instant Answer + Purpose)
- [ ] Description contains a valid Instant Answer Page Link (e.g. https://duck.co/ia/view/my_ia)

**Instant Answer Page** (for new Instant Answers)
- [ ] Instant Answer page is correctly filled out and contains:
    - The **Title** is appropriately named and formatted
    - The **IA topics** are present and appropriate
    - The **Description** is clear and coherent
    - **Source Name** exists if applicable
    - All **Example Queries** trigger on [Beta](https://beta.duckduckgo.com/)

**Code**
- [ ] Adheres to the [DuckDuckGo Style Guide](https://docs.duckduckhack.com/resources/code-style-guide.html)
- [ ] Behaviour is appropriately tested. If improvement, tests are adequately extended.
- [ ] There is no unnecessary files in place (such as editor config files)
- [ ] There is no API keys / secrets present
- [ ] Tests are passing (run `$ duckpan test <fathead_id>`)
    - Tester should report any failures

**Ready to merge?**

- [ ] Has this IA been deployed to and tested on [beta.duckduckgo.com](https://beta.duckduckgo.com/)?
- [ ] For larger commits, has this been approved be more than one community member?
- [ ] The number of reviews is appropriate for this type of PR
- [ ] The commit message is clear, coherent and fitting

**Pull Request Review Guidelines**: https://docs.duckduckhack.com/programming-mission/pr-review.html

---

Instant Answer Page: https://duck.co/ia/view/gitkraken

### CODE REVIEW

**Comment by daxtheduck** [comment]
Metadata placeholder post
            
*If you haven't already, please edit your first comment to include a link to the IA page at the bottom.*
            
---
            
**How to find your IA page**
- New Instant Answer
  -- follow the steps [here](https://duck.co/ia/new_ia) to create your IA page.
              
- Existing Instant Answer
  -- find your IA page from this [list](https://duck.co/ia)

File: share/goodie/cheat_sheets/json/gitkraken.json [hunk 1]
```diff
@@ -0,0 +1,113 @@
+{
+    "id": "gitkraken_cheat_sheet",
+    "name": "Gitkraken",
+    "description": "GitKraken Shortcuts",
+
```
Comment (ManrajGrover) [inline share/goodie/cheat_sheets/json/gitkraken.json]: Nit: No blank lines please apart from one at the end of source.

File: share/goodie/cheat_sheets/json/gitkraken.json [hunk 2]
```diff
@@ -0,0 +1,113 @@
...
+            },
+            {
+                "key": "[⌘] [↑]",
+                "val": "Select First Item (Mac)"
+            },
+            {
+                "key": "[Ctrl] [Home]",
+                "val": "Select First Item (Windows/Linux)"
+            },
+                        {
```
Comment (ManrajGrover) [inline share/goodie/cheat_sheets/json/gitkraken.json]: Nit: Please align this bracket.

File: share/goodie/cheat_sheets/json/gitkraken.json (hunk 1 above)
Comment (ManrajGrover) [inline share/goodie/cheat_sheets/json/gitkraken.json]: Please fix this in all places

File: share/goodie/cheat_sheets/json/gitkraken.json [hunk 3]
```diff
@@ -0,0 +1,113 @@
+{
+    "id": "gitkraken_cheat_sheet",
+    "name": "Gitkraken",
+    "description": "GitKraken Shortcuts",
```
Comment (ManrajGrover) [inline share/goodie/cheat_sheets/json/gitkraken.json]: Let's make description as `Keyboard shortcuts for GitKraken, a Git GUI for Windows, Mac and Linux`

**Comment by ManrajGrover** [comment]
@pjhampton The `keyboard` template seems to break with long shortcuts. I think we should fix it.

File: share/goodie/cheat_sheets/json/gitkraken.json (hunk 1 above)
Comment (sahildua2305) [inline share/goodie/cheat_sheets/json/gitkraken.json]: @ManrajGrover isn't it better to keep one blank line at the end of a section to enhance the readability of the JSON code? Just a thought, not an opinion.

File: share/goodie/cheat_sheets/json/gitkraken.json [hunk 4]
```diff
@@ -0,0 +1,113 @@
...
+
+    "metadata": {
+        "sourceName": "GitKraken",
+        "sourceUrl" : "https://www.gitkraken.com/downloads/gitkraken-cheat-sheet-1mar2017.pdf"
+    },
+
+    "aliases": [
+        "git kraken"
+    ],
```
Comment (sahildua2305) [inline share/goodie/cheat_sheets/json/gitkraken.json]: @mustang25 you may add `gitkraken` as an alias as well.

File: share/goodie/cheat_sheets/json/gitkraken.json (hunk 1 above)
Comment (ManrajGrover) [inline share/goodie/cheat_sheets/json/gitkraken.json]: @sahildua2305 Keeping consistency throughout cheat sheets in mind, I would say we should follow either one or the other.

**Comment by pjhampton** [comment]
@ManrajGrover agreed! Let's not let that hold up this PR though :)

**Comment by mintsoft** [comment]
I'm going to close this for now (doing some house-keeping). Feel free to ping and I'll re-open if I've closed it in error

**Pull request closed without merging**